### PR TITLE
[CodeClean] remove unnecessary code

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. remove unnecessary function to get caps from tensor-info.
2. move to proper build option and cleanup unnecessary option.

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -328,14 +332,15 @@ CFLAGS="${CFLAGS} -fprofile-arcs -ftest-coverage"
...
-	%{enable_python} build
+	--bindir=%{nnstexampledir} --includedir=%{_includedir} -Dinstall-example=true \
+	%{enable_api} %{enable_tizen} %{element_restriction} -Denable-env-var=false -Denable-symbolic-link=false \
+	-Denable-pytorch=false -Denable-caffe2=false %{enable_tf} %{enable_python} \
+	%{enable_nnfw_runtime} %{enable_mvncsdk2} %{enable_armnn} %{enable_edgetpu} \
+	build
 
 ninja -C build %{?_smp_mflags}
 
+%if 0%{?unit_test}
```
Comment (dongju-chae) [inline packaging/nnstreamer.spec]: I remember that some unittests should be tested unconditionally in Tizen.
Refer to discussions in https://github.com/nnsuite/nnstreamer/pull/1976.

Comment (jaeyun-jung) [reply packaging/nnstreamer.spec]: Question & Discussion: nnfw and tizen-sensor should always run even if unit-test is not enabled?

File: packaging/nnstreamer.spec [hunk 2]
```diff
@@ -345,11 +350,12 @@ export NNSTREAMER_DECODERS=$(pwd)/build/ext/nnstreamer/tensor_decoder
 %define test_script $(pwd)/packaging/run_unittests_binaries.sh
 
 %if %{with tizen}
+    %ifarch %arm aarch64
```
Comment (kparichay) [inline packaging/nnstreamer.spec]: This is not needed. The test_script handles if the file exists or not.

File: packaging/nnstreamer.spec (hunk 1 above)
Comment (myungjoo) [reply packaging/nnstreamer.spec]: Requirements & Background:
1. Unit test is disabled for arm/arm64 in CI server because it takes too much time.
2. NNFW is enabled for arm/arm64 only.
3. Unit test for nnfw only won't take too much time.
4. Tizen-sensor has a lot of dependencies on Tizen; thus, I'm worried about regressions from changes in Tizen.org. ==> Do unit-test Tizen-sensor in tizen.org's infrastructure unconditionally. But not for other unit tests in tizen.org's infrastructure (takes too much time)
5. NNFW has some dependencies on Tizen; thus, I'm also worried about regressions from changes in Tizen.org, as well.

In short, for NNFW & Tizen-sensor, regressions from changes in Tizen.org might haunt us. With unit-test (nnfw/tizen-sensor) enabled unconditionally in Tizen, we can be relieved without the high cost of all-unit-test-ARM.

File: packaging/nnstreamer.spec (hunk 2 above)
Comment (jaeyun-jung) [reply packaging/nnstreamer.spec]: updated

File: packaging/nnstreamer.spec (hunk 1 above)
Comment (jaeyun-jung) [reply packaging/nnstreamer.spec]: I understand, rollback to old.
